h2o-docs/src/product/grid-search.rst

```diff
@@ -294,6 +297,138 @@ Random Grid Search Examples
+
+.. tabs::
+  .. code-tab:: r R
+
+    # Train and validate a cartesian grid of GBMs
+    gbm_grid1 <- h2o.grid("gbm", x = x, 
+                          y = y, grid_id = "gbm_grid_test", 
+                          training_frame = train, 
+                          validation_frame = valid, 
+                          ntrees = 100, seed = 1, 
+                          hyper_params = gbm_params1, 
+                          export_checkpoints_dir = tempdir())
+
+    # Identify the grid_id and model_ids
+    grid_id <- gbm_grid1@grid_id
+    gbm_grid_model_count <- length(gbm_grid1@model_ids)
+
+    # Wipe the cloud to simulate cluster restart 
+    #(the models will no longer be available)
+    h2o.removeAll()
+
+    # Retrieve the saved grid
+    grid <- h2o.loadGrid(paste0(tempdir(), "/", grid_id))
+    grid
+
+
+  .. code-tab:: python
+
+    # Add the save location
+    checkpoints_dir = tempfile.mkdtemp()
+
+    # Train and validate a cartesian grid of GBMs
+    gbm_grid1 = H2OGridSearch(model=H2OGradientBoostingEstimator, 
+                              grid_id='gbm_grid', 
+                              hyper_params=gbm_params1, 
+                              export_checkpoints_dir=checkpoints_dir)
+    gbm_grid1.train(x=x, y=y, 
+                    training_frame=train, 
+                    validation_frame=valid, 
+                    ntrees=100, 
+                    seed=1)
+
+    # Identify the grid_id and model_ids
+    grid_id = gbm_grid1.grid_id
+    old_grid_model_count = len(gbm_grid1.model_ids)
+
+    # Wipe the cloud to simulate cluster restart 
+    #(the models will no longer be available)
+    h2o.remove_all()
+
+    # Retrieve the saved grid
+    grid = h2o.load_grid(checkpoints_dir + "/" + grid_id)
+    grid
+
+
+Manual Export Example
+'''''''''''''''''''''
+
+Using the `Grid Search <#grid-search-examples>`__ example through the hyperparameters section, run the following additional commands to retrieve the manually exported saved grid.
+
+.. tabs::
+  .. code-tab:: r R
+
+    # Train and validate a cartesian grid of GBMs
+    gbm_grid1 <- h2o.grid("gbm", x = x, 
+                          y = y, grid_id = "gbm_grid1", 
+                          training_frame = train, 
+                          validation_frame = valid, 
+                          ntrees = 100, seed = 1, 
+                          hyper_params = gbm_params1)
+
+    # Identify the grid_id and model_ids
+    grid_id <- gbm_grid1@grid_id
+    gbm_grid1_model_count <- length(gbm_grid1@model_ids)
+
+    # Save the grid
+    saved_path <- h2o.saveGrid(grid_directory = tempdir(), grid_id = grid_id)
+
+    # Wipe the cloud to simulate cluster restart 
+    #(the models will no longer be available)
+    h2o.removeAll()
+
+    # Retrieve the saved grid
+    grid <- h2o.loadGrid(saved_path)
+    grid
+
+
+  .. code-tab:: python
+
+    # Add the save location
```

Comment:
Same comment here. Add `import tempfile` because users will likely run one example or the other.